src/plugin/file/csv/csvdescriptor.js

```diff
@@ -181,6 +181,8 @@ plugin.file.csv.CSVDescriptor.createFromConfig = function(config) {
  */
 plugin.file.csv.CSVDescriptor.updateFromConfig = function(descriptor, config) {
   descriptor.setColor(config['color']);
+  descriptor.setIcon(config['icon']);
+  descriptor.setShapeName(config['shapeName']);
```

Comment:
I know this is static, but since the options step is available on nearly all import types, common items like this should be consolidated in another method and called from each descriptor type.

Reply:
Interesting ... there's a lot of repeated code across those import types. Seems static functions can still be inherited, I'll try it out.

Reply:
Yes, SHP supports all geometry types including Point/MultiPoint.

Reply:
> Interesting ... there's a lot of repeated code across those import types. Seems that static functions can still be inherited, I'll try it out.

To be clear, static functions are not inherited. You're just consolidating repeated logic to a function and calling it from those places instead.